frontend/src/store/index.ts

```diff
@@ -12,19 +13,19 @@ export type Registry = {
 };
 
 export type ContextRegistry = {
-  getRegistry: () => Registry;
+  getRegistry: () => ReducerRegistry<any>;
 };
 
 export const RegistryContext = createContext<ContextRegistry>({} as ContextRegistry);
 
-let registry: Registry;
+let registry: ReducerRegistry<any>;
 
-export function init(...middleware: Middleware[]): Registry {
+export function init(...middleware: Middleware[]): ReducerRegistry<any> {
   registry = getRegistry(
     {},
     [thunk, promiseMiddleware, notificationsMiddleware({ errorDescriptionKey: ['detail', 'stack'] }), ...middleware.filter(Boolean)],
     undefined,
   );
-  registry.register(SDKReducers);
+  registry.register(SDKReducers as any);
```

Comment:
I'm curious as to why we need to cast to any here. Do we need a type change in the SDK?

Reply:
There was a type mismatch between what SDK was providing vs what `ReducerRegistry` expected.